### PR TITLE
Update UG, PPP, and AboutUs

### PR DESCRIPTION
**Pull request by ThisisXXZ**
Main changes include:
* clarify the descriptions of ``stats`` command in UG and DG
* add some FAQ in UG
* fix some "FindMySEP" naming mistakes
* update PPP and AboutUs

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -282,23 +286,27 @@ Adios, amigo!
 >
 > A: FindOurSEP is an open-source application, and we welcome developers to share their ideas. You may find the source code on [GitHub](https://github.com/AY2425S1-CS2113-W12-2/tp).
 
+> **Q: How does FindOurSEP decide who should be allocated first?**
+>
+> A: In FindOurSEP, students are prioritized based on their GPA, with higher GPAs sorted first. When two students share the same GPA, priority is given to the one listed earlier, following a first-come-first-serve basis.
+
```
Comment (isaacsaw25) [inline docs/UserGuide.md:292]: This is wonderful to have in the FAQ! 😃

File: docs/images/thisisxxz.jpg
Comment (isaacsaw25) [inline docs/images/thisisxxz.jpg:1]: Great PFP! 😅

**Review by isaacsaw25 — APPROVED** [review]
LWTM! 😄

**Review by ehz0ah — APPROVED** [review]
ACK

**Pull request merged**